src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -9314,9 +9314,9 @@ def test_aks_create_with_nsg_control(self, resource_group, resource_group_locati
         asg_ids = [asg1['id'], asg2['id']]
 
         self.kwargs.update({
-            'asg1_id': asg1['id'],
-            'asg2_id': asg2['id'],
+            'asg_ids_argument': '--nodepool-asg-ids {}'.format(' '.join(asg_ids)),
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83672330&view=results) to validate the change.